lib/chef/util/windows/net_user.rb

```diff
@@ -96,8 +96,13 @@ def validate_credentials(passwd)
     token = Security.logon_user(@username, nil, passwd,
                LOGON32_LOGON_NETWORK, LOGON32_PROVIDER_DEFAULT)
     true
-  rescue Chef::Exceptions::Win32APIError
-    false
+  rescue Chef::Exceptions::Win32APIError => e
+    if e.to_s.include? "The user name or password is incorrect"
```

Comment:
Are these error strings localized? What's the error code that's raised? That should be printed as well by `Chef::Exceptions::Win32APIError` and wouldn't change. Is it 1326 / ERROR_LOGON_FAILURE?. 

It is too bad that `Chef::ReservedNames::Win32::Error.raise!` doesn't pass the code such that you could just check `e.code`.

Reply:
Right, so I've extended Win32APIError so you can now check `e.code` and not have to deal with localization.